lib/geoblacklight/download.rb

```diff
@@ -33,14 +33,18 @@ def get
       end
     end
 
+    def reveal_mimetype(download)
```

Comment:
Could this be a private method? And I wonder if you could put all of the logic in a method like `matches_mimetype`?

```ruby
if matches_mimetype(download)
  file.write download.body
...

private
def matches_mimetype(download)
  MIME::Type.simplified(download.headers['content-type']) == @options[:content_type]
end
```

You may not be able to unit test this method, but you could add a case to the `#create_download` block?

Just thinking here about limiting our exposure of the public api. Would someone need to modify this method?